projects/network_tester/src/ui_task.c

```diff
@@ -83,6 +87,13 @@ typedef struct
     uint32_t    beep_off_time;
     uint32_t    num_cycles;
 } beep_t;
+
+typedef enum
+{
+    unset   = 0,
```

Comment:
Enum variants should be all uppercase, since they're constants. Also, since all enum variants share a namespace, these values should have some common prefix. Something like `LL_NT_STATE_UNSET`.